README.md

```diff
@@ -6,6 +6,25 @@
 
 > This is a preview version of the updated React Native Tools extension. It's not recommended to use the preview version and the stable one at the same time.
 
+## React Native Tools Preview migration guide
+
+In the Preview version of the extension we changed debug types in debug configurations. The table below shows the difference between debug configuration of the stable and the preview version.
+
+|RNT stable|RNT preview|
```

Comment:
I am also thinking that we could add some image to demonstrate how the old configuration becomes the preview one in addition to this table.